Duality/Resources/RenderTarget.cs

```diff
@@ -211,6 +211,16 @@ protected void FreeNativeRes()
 		/// </summary>
 		protected void SetupNativeRes()
 		{
+			foreach (var target in this.targets.Where(t => t != null).Res())
+			{
+				if (target.PixelWidth == 0 || target.PixelHeight == 0)
+				{
+					Log.Core.WriteError("Error initializing RenderTarget {0}: Texture {1} has a dimension of 0 {2}",
+						this.FullName, target.FullName, Environment.NewLine);

```

Comment:
- No need to add a NewLine at the end of a log. The log system will do for you.
- Add single quotes (`'`) around `{0}` and `{1}` as they are referring to Resources.
- Instead of passing the `FullName` property, pass the Resource instance directly (`this` / `target`). The Log will display them using their full name anyway and you get double-click ability in the editor's LogView as a bonus.